gradle/mockito-core/javadoc.gradle

```diff
@@ -1,7 +1,19 @@
 //It seems the gradle javadoc task works file by file and as such disable some features of javadoc tool
 //such as link to packages, https://groups.google.com/d/msg/gradle-dev/R83dy_6PHMc/bgw0cUTMFAAJ
+
+// In this directory we keep two copies of the `package-list` file for java 8 api docs:
+// - as `package-list` for Java 9+
+// - as `element-list` for java 8
+def java8DocsDir = "$buildDir/java-8-docs"
+
+task copyJava8Docs(type: Copy) {
```

Comment:
Rather than copying, can we use the source directory for `linksoffline`? That should be `$projectDir/gradle/mockito-core`